uint32 cameraId 				#index to the camera whose module wants image (see Camera Indexes)
uint32[] cameraParameterIds 	#Vector of Ids of the camera parameters (see Camera parameters)
uint32[] newValues 				#Parameterâ€™s new values.
---
bool[] isSetList
